src/full/Agda/TypeChecking/Primitive/Cubical.hs

```diff
@@ -1256,12 +1279,24 @@ toCType ty = do
   sort <- reduce $ getSort ty
   case sort of
     Type l -> return $ Just $ LType (LEl l (unEl ty))
-    SSet{} -> do
+    SSet _ -> do
       t <- reduce (unEl ty)
       case t of
         Def q [] -> return $ Just $ ClosedType sort q
         _        -> return $ Nothing
-    _      -> return $ Nothing
+    Prop _       -> pure Nothing
+    Inf _ _      -> pure Nothing
+    IntervalUniv -> pure Nothing  -- ClosedType?  (See comment on CType.)
+    LevelUniv    -> pure Nothing  -- TODO: Or some ClosedType?
```

Comment:
correct?

Reply:
Oh, actually `IntervalUniv` should have the check that's at `SSet` atm. I suppose that got overlooked when `IntervalUniv` got introduced. OTOH it seems then that the whole `ClosedType` thing might not actually be needed then.

I think it was meant to support path constructors for HITs, but those needed special care anyway?

Reply:
I raised this as #7542.